projects/frontend/data-pipelines/gui/projects/data-pipelines/tsconfig.lib.json

```diff
@@ -0,0 +1,26 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
```

Comment:
These caused some problems for me in the pre-commit checks. Or were they excluded from the checks somehow?

Reply:
Didn't catch that, could you give an example?
The pre-commit hooks are evaluated here https://results.pre-commit.ci/run/github/387724608/1676309070.Y5DNitheSAuTqTJMdjFSyA

Reply:
My bad, you ignored those in the previous PR.